setup.py

```diff
@@ -23,7 +23,7 @@
 
     include_package_data=True,
 
-    python_requires='>=2.7',
+    python_requires='>=3.5',
```

Comment:
have we documented this for new hire onboard? not necessarily a problem, but i think we've had plenty of py2 support before. requiring 3 for docker will require 3 for pretty much all devs as we move docker builds into regular mvn builds. we should have a clean, easy, setup path to ensure this works well.

Reply:
Doesn't look like the new hire doc covers Python at all, but the Release CLI setup doc does require 3.

This only affects integration testing for this module, so it's not a global change. I wanted to go ahead and make the jump to 3 here since there were no existing tests to speak of. The newer unittest module is also nice to have.

Reply:
This did have unintended side effects due to how the package gets installed with setuptools, so I've set it back to 2 for now.

Reply:
ack. in general i'm fine with requiring 3 (even if i still default to using print as a statement :)), just wanted to make sure people are setup for it.